pyartcd/pyartcd/jenkins.py

```diff
@@ -175,17 +176,19 @@ def is_build_running(build_path: str) -> bool:
     """
 
     init_jenkins()
-    jenkins_url = get_jenkins_url()
-    response = requests.get(f'{jenkins_url}/{build_path}/api/json')
-    if response.status_code != 200:
-        logger.info('Could not fetch data for build %s', build_path)
-        raise ValueError
-
-    build_data = response.json()
-    logger.info('Build %s %s in progress',
-                 f'{constants.JENKINS_UI_URL}/{build_path}',
-                 'is' if build_data['inProgress'] else 'is not')
-    return build_data['inProgress']
+    job_path, build_number = build_path.rstrip("/").rsplit("/", 1)
+    job_name = job_path.rsplit("/", 1)[1]
+    job_url = jenkins_client.base_server_url() + "/" + job_path
+    try:
+        job = Job(job_url, job_name, jenkins_client)
+    except requests.exceptions.HTTPError as err:
```

Comment:
we should raise error instead of ignore it as False
and no need to split into two steps to catch error

Reply:
done

Reply:
Did this part changed?